playbook-website/app/javascript/site_styles/_changelog.scss

```diff
@@ -9,4 +9,8 @@
       margin-top: 0;
     }
   }
+
+  .anchorjs-link {
+    font-size: 1.25em !important;
+  }
```

Comment:
Why not use our tokens? Can we loose the important?

Reply:
I'll use our tokens, but currently when i get rid of !important, my styles get overwritten by inline styles.